### PR TITLE
Further CI support improvements.

### PR DESCRIPTION
**Pull request by stefanseefeld**
* Fix MSVC build failures
* Robustify conda environment setup

(This isn't entirely ready for review / merge yet. I expect to add another commit once a new conda build package is released.)

### CODE REVIEW

**Comment by pitrou** [comment]
Do you think the AppVeyor test script can use this as well? (note it uses `--tags important` to shorten the test time)

**Comment by stefanseefeld** [comment]
That would be ideal indeed. Perhaps we can try setting it up to work from the branch, to test it.

**Comment by dhirschfeld** [comment]
Is the Windows CI down at the moment? The last Windows build appears to be `0.26.0.dev np110py35_53` whereas  the latest build number for linux is `185`.

Will this PR fix that? I'm very keen to try out both the new linalg support and the `cfunc` support!

**Comment by pitrou** [comment]
> Perhaps we can try setting it up to work from the branch, to test it

AppVeyor is supposed to run on every PR, perhaps something wrong went with the latest changesets.

**Comment by pitrou** [comment]
"Error parsing appveyor.yml: (Line: 14, Col: 21, Idx: 557) - (Line: 14, Col: 25, Idx: 561): While parsing a block mapping, did not find expected key."

File: numba/_lapack.c [hunk 1]
```diff
@@ -1,9 +1,4 @@
 /*
- * This file contains wrappers of BLAS and LAPACK functions
```
Comment (stuartarchibald) [inline numba/_lapack.c]: Why is this comment deleted?

File: numba/_lapack.c [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
- */
-#include <complex.h>
```
Comment (stuartarchibald) [inline numba/_lapack.c]: NOTE: potential fix for https://github.com/numba/numba/issues/1876

File: numba/_lapack.c (hunk 1 above)
Comment (stefanseefeld) [inline numba/_lapack.c]: Sorry, for some reason I (admittedly mindlessly) thought that comment applied to the following <complex.h> inclusion.

**Comment by stefanseefeld** [comment]
I tried to get the .appveyor.yml syntax more in line with .travis.yml. Obviously I failed, so I have reverted that bit of environment configuration logic.

**Comment by seibert** [comment]
@dhirschfeld: That issue was due to a problem in the Win64 builder that was fixed yesterday.  I think the latest Win64 packages are now available on anaconda.org.

**Comment by dhirschfeld** [comment]
Thanks, I've updated and have been playing around with the latest functionality!

**Pull request merged**